src/server/index.js

```diff
@@ -33,6 +34,9 @@ setupAuth0Passport()
 if (!process.env.SUPPRESS_SEED_CALLS) {
   seedZipCodes()
 }
+if (!process.env.SUPPRESS_MIGRATIONS) {
```

Comment:
no because I don't think it should be the default.  We could add it to docs somewhere, I suppose